setup.py

```diff
@@ -8,6 +8,8 @@
 
 deps = [
     'marionette-client == 0.9',
+    'mozinstall == 1.11',
+    'mozlog == 2.10',
```

Comment:
I was told to include other packages as dependencies if we make use of those ourselves. Even when there is a dependency set for that other package by a package we also depend on. I'm really confused to see those two different feedback. So what is correct?